### ISSUE TITLE
Deprecation warnings on Django 3.1a1

### ISSUE THREAD

**Issue by jannh** [mbi/django-simple-captcha#186]
When using django-simple-captcha with the latest Django 3.1a1, the following deprecation warning appears:

´´´
  warnings.warn(
/home/jannh/.virtualenvs/siam/lib/python3.8/site-packages/django/conf/urls/__init__.py:16: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
´´´

The recommendation from Django is to use django.urls.path (or re_path, which provides the same functionality like url previously). This has however only been added in Django 2.0 - what do you think about dropping support for Django < 2.0 (and therefore also python2) and moving to the new syntax? I could contribute a PR with the necessary changes and cleanup of the six stuff.

**Comment by TheBuky** [comment]
If I can take advantage of this issue to point out an other deprecation warning:

```python
/home/user/.virtualenvs/django/lib/python3.7/site-packages/captcha/fields.py:189: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  kwargs['error_messages'].update({'invalid': ugettext_lazy('Invalid CAPTCHA')})
```

You need to replace **ugettext_lazy** in [fields.py](https://github.com/mbi/django-simple-captcha/blob/c7d411f11dd94e81b8968af7c0cec4c89c6c3e32/captcha/fields.py#L228) by [gettext_lazy](https://docs.djangoproject.com/en/dev/ref/utils/#django.utils.translation.gettext_lazy).

**Comment by mbi** [comment]
Fixed via 8a60838, thanks!
